src/status_im/contexts/profile/contact/block_contact/view.cljs

```diff
@@ -0,0 +1,70 @@
+(ns status-im.contexts.profile.contact.block-contact.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [status-im.constants :as constants]
+            [status-im.contexts.profile.contact.block-contact.style :as style]
+            [status-im.contexts.profile.utils :as profile.utils]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn on-close [] (rf/dispatch [:hide-bottom-sheet]))
+
+(defn view
+  []
+  (let [{:keys [customization-color contact-request-state public-key]
+         :as   contact}                       (rf/sub [:contacts/current-contact])
+        ;; TODO(@mohsen): remove :blue, https://github.com/status-im/status-mobile/issues/18733
+        customization-color                   (or customization-color :blue)
```

Comment:
I think we have a constant for the default profile color. Could we use the same here?

Reply:
didn't know that, just applied. thanks